packages/spruce-skill-server/tests/v1APIMocks.js

```diff
@@ -0,0 +1,36 @@
+const config = require('config')
+
+module.exports = ctx => ({
+	async get(path, query) {
+		const matches = path.match(/\/locations\/([^/]+)\/users\/([^/]+)/)
+		if (matches && matches[1] && matches[2]) {
+			// ctx.sb.user() has been called. Fetch the data from the DB. Used for v1 authentication
+			const locationId = matches[1]
```

Comment:
Should we consider returning errors to the implementer by doing something like a `typeof` check on these id's? Or coerce with `parseInt()` or `parseFloat()` and check for `NaN`?

Reply:
I'm not really following what you mean. Ids in our system are UUIDs (strings). This part of the code is just intercepting the `ctx.sb.user()` call and looking up the user for use in our tests (so we don't rely on any external api calls for tests).

Reply:
ahh, i thought it was for a mock server that we would be building requests against. additionally, good to know about the UUIDs. marking the conversation as resolved.